unit-test/fm_cmd_utils_tests.c

```diff
@@ -154,159 +154,125 @@ void Test_FM_VerifyOverwrite_OverwriteIsFalse(void)
 }
 
 /* **************************
- * LoadOpenFileData
+ * GetOpenFilesData Tests
  * *************************/
-/*
-void Test_LoadOpenFileData_OpenFileCountUnchanged(void)
+void Test_FM_GetOpenFilesData_OSForEachObjectCalled(void)
 {
     // Arrange
-    uint32 dummy_id = 333;
-    void * dummy_args = NULL;
-
-    uint32 count_openfile_before = OpenFileCount;
+    FM_OpenFilesEntry_t *dummy_files_data = NULL;
 
     // Act
-    LoadOpenFileData(dummy_id, dummy_args);
-
-    uint32 count_openfile_after = OpenFileCount;
+    uint32 open_file_count = FM_GetOpenFilesData(dummy_files_data);
 
     // Assert
-    UtAssert_True(count_openfile_after == count_openfile_before,
-                    "Openfile value is %u and should be %u",count_openfile_after,count_openfile_before);
+    UtAssert_True(open_file_count == 0, "OpenFileCount is %u and should be 0", open_file_count);
```

Comment:
Just UtAssert_UINT32_EQ(FM_GetOpenFilesData(dummy_files_data), 0); and remove the unnecessary local open_file_count.